vega_sim/api/market.py

```diff
@@ -169,6 +192,12 @@ def build(self):
     def set(self, parameter, value):
         rsetattr(self, attr=parameter, val=value)
 
+    def is_future(self) -> bool:
+        return self.instrument.future != None
+
+    def is_perp(self) -> bool:
+        return self.instrument.perp != None
```

Comment:
```suggestion
        return self.instrument.perp is not None
```

http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html

Reply:
Amended now, thanks a lot.